test/browser-plugins/plugins.test.ts

```diff
@@ -84,7 +83,6 @@ const runPluginTest = <
             const context = plugin.createLaunchContext();
             browser = await plugin.launch(context as never);
 
-            expect(fs.existsSync(context.userDataDir)).toBeTruthy();
```

Comment:
When we remove this line, shouldn't we remove the whole test?

Reply:
fixed, thanks